python/lsst/validate/drp/check.py

```diff
@@ -181,15 +233,80 @@ def checkPhotometry(snr, mag, mmagrms, dist, match,
           (np.median(mmagrms), "mmag"))
 
     bright = np.where(np.asarray(snr) > brightSnr)
-    photoScatter = np.median(np.asarray(mmagrms)[bright])
+    photScatter = np.median(np.asarray(mmagrms)[bright])
     print("Photometric scatter (median) - SNR > %.1f : %.1f %s" %
-          (brightSnr, photoScatter, "mmag"))
+          (brightSnr, photScatter, "mmag"))
+
+    fit_params = fitPhotErrModel(mag[bright], mmagErr[bright])
 
-    if photoScatter > medianRef:
+    if photScatter > medianRef:
         print("Median photometric scatter %.3f %s is larger than reference : %.3f %s "
-              % (photoScatter, "mmag", medianRef, "mag"))
+              % (photScatter, "mmag", medianRef, "mag"))
     if match < matchRef:
         print("Number of matched sources %d is too small (shoud be > %d)"
               % (match, matchRef))
 
-    return photoScatter
+    return pipeBase.Struct(name="checkPhotometry",
+                           photRmsScatter=photScatter, 
+                           photErrModel=photErrModel.__doc__,
+                           photErrFitParams=fit_params)
+
+
+def astromErrModel(snr, theta=1, sigma_sys=0.01, C=1):
+    """Calculate expected astrometric uncertainty based on SNR.
+
+    mas = C*theta/SNR
```

Comment:
Interesting call whether math like this should be written ascii-style for clarity to a code reader to rendered as LaTeX (below) for a future doc reader. We don't have a policy on this, so no need to change this yet.

```
.. math:: \sigma_\mathrm{astrom} = \frac{C \theta}{\mathcal{S/N}}
```